src/router.tsx

```diff
@@ -125,6 +126,9 @@ function ChildrenOrError(props: { children: JSX.Element }) {
                     password={state.password}
                 />
             </Match>
+            <Match when={state.network_status?.connectionType === "none"}>
+                <NoConnection />
+            </Match>
```

Comment:
this should probably be before `setup_error`. having no internet would likely be the cause of a setup error

Reply:
One of the first things that gets returned from the setup phase is the LNURL call not working

Reply:
I'm not sure if you guys are agreeing or disagreeing here.

Reply:
agreeing

Reply:
okay rebased and fixed